examples/planning/forecast_reach.py

```diff
@@ -0,0 +1,277 @@
+                point.forecast.total_reach.value,
+                point.forecast.total_impressions.value,
+                product_splits])
+
+
+def forecast_manual_mix(
+        client, customer_id, location_id, currency_code, budget):
+    """Pulls a forecast for product mix created manually.
+
+    Args:
+      client: A google.ads.google_ads.client.GoogleAdsClient instance.
+      customer_id: The customer ID for the reach forecast.
+      location_id: Location ID returned by ListPlannableLocations.
+      currency_code: Three-character ISO 4217 currency code.
+      budget: Budget to allocate to the plan.
+    """
+    product_mix = []
+    product_splits = [('TRUEVIEW_IN_STREAM', 0.15), ('BUMPER', 0.85)]
+    for product, split in product_splits:
+        planned_product = client.get_type('PlannedProduct', version='v2')
+        planned_product.plannable_product_code.value = product
+        planned_product.budget_micros.value = math.trunc(budget * 1.0e6 * split)
+        product_mix.append(planned_product)
+
+    _request_reach_curve(
+            client, customer_id, product_mix, location_id, currency_code)
+
+
+def forecast_suggested_mix(
+    client, customer_id, location_id, currency_code, budget):
+    """Pulls a forecast for a product mix based on your set of preferences.
+
+    Args:
+      client: A google.ads.google_ads.client.GoogleAdsClient instance.
+      customer_id: The customer ID for the reach forecast.
+      location_id: Location ID returned by ListPlannableLocations.
+      currency_code: Three-character ISO 4217 currency code.
+      budget: Budget to allocate to the plan.
+    """
+    preferences = client.get_type('Preferences', version='v2')
+    preferences.has_guaranteed_price.value = True
+    preferences.starts_with_sound.value = True
+    preferences.is_skippable.value = False
+    preferences.top_content_only.value = True
+    preferences.ad_length = client.get_type(
+            'ReachPlanAdLengthEnum', version='v2').FIFTEEN_OR_TWENTY_SECONDS
+
+    reach_plan_service = client.get_service('ReachPlanService', version='v2')
+    mix_response = reach_plan_service.generate_product_mix_ideas(
+            customer_id=customer_id,
+            plannable_location_id=_string_value(client, location_id),
+            preferences=preferences,
+            currency_code=_string_value(client, currency_code),
+            budget_micros=_int_64_value(client, budget * 1.0e6))
+
+    product_mix = []
+    for product in mix_response.product_allocation:
+        planned_product = client.get_type('PlannedProduct', version='v2')
+        planned_product.plannable_product_code.value = (
+                product.plannable_product_code.value)
+        planned_product.budget_micros.value = product.budget_micros.value
+        product_mix.append(planned_product)
+
+    _request_reach_curve(
+            client, customer_id, product_mix, location_id, currency_code)
+
+
+def main(client, customer_id):
+    location_id = '2840'  # US
```

Comment:
suggest you add link to reference list of location IDs, ie. https://developers.google.com/google-ads/api/reference/rpc/google.ads.googleads.v2.services#google.ads.googleads.v2.services.GeoTargetConstantService

Reply:
Done -- Added this link and a reference to the `ListPlannableLocations` method in the `ReachPlanService` docs. Also updated the docstrings to match.